Documentation/rkt-vs-other-projects.md

```diff
@@ -0,0 +1,122 @@
+* [rkt vs machinectl](#rkt-vs-machinectl)
+* [rkt vs qemu-kvm, lkvm](#rkt-vs-qemu-kvm-lkvm)
+
+## rkt vs Docker
+
+The Docker Engine, or simply Docker, is an application container runtime system that runs as a centralized API daemon. Docker can take `a "Docker Image" name, such as `quay.io/coreos/etcd`, and download, execute, and monitor the application.
+Functionally, this is all similar to rkt; however, along with "Docker Images", rkt can also download and run "Application Container Images" (ACIs) specified by the [App Container Specification](https://github.com/appc/spec) (appc).
+
+Besides also supporting ACIs, rkt has a substantially different architecture that is designed with composability and security in mind.
+
+#### Process Model
+
+At the time of writing, the Docker Engine daemon downloads container images, launches container processes, exposes a remote API, and acts as a log collection daemon, all in a centralized process running as root.
+While such a centralized architecture is convenient for deployment, it does not follow best practices for Unix process and privilege separation; further, it makes Docker difficult to properly integrate with Linux init systems such as upstart and systemd.
+Since running a Docker container from the command line (i.e. using `docker run`) just talks to the Docker daemon API, which is in turn responsible for creating the container, init systems are unable to directly track the life of the actual container process.
+
+rkt has no centralized "init" daemon, instead launching containers directly from client commands, making it compatible with init systems such as systemd, upstart, and others.
+
+![rkt-vs-docker-process-model](rkt-vs-docker-process-model.png)
+
+
+#### Privilege Separation
+
+rkt uses standard Unix group permissions to allow privilege separation between different operations.
+Once the rkt data directory is correctly set up (for example, by `rkt install`), the container image downloads and signature verification can run as a non-privileged user.
+
+![rkt-vs-docker-fetch](rkt-vs-docker-fetch.png)
+
+
+## rkt vs runC
+
+[runC][runc-github] is a low-level container runtime and an implementation of the [Open Container Initiative specification][oci-spec-github].
+runC exposes and expects a user to understand low-level details of the host operating system and configuration.
+It requires the user to separately download or cryptographically verify container images, and for "higher level tools" to prepare the container filesystem.
+runC does not have a centralized daemon, and, given a properly configured "OCI bundle", can be integrated with init systems such as upstart and systemd.
+
+rkt includes the same functionality as runC but does not expect a user to understand low-level details of the operating system to use, and can be invoked as simply as `rkt run coreos.com/etcd,version=v2.2.0`.
+It can download both "Docker Images" and "Application Container Images".
```

Comment:
It can download both [Docker Images](https://github.com/docker/docker/blob/master/image/spec/v1.md) and [Application Container Images](https://github.com/appc/spec/).